manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml

```diff
@@ -37,3 +37,23 @@ spec:
       selfHeal: true
     syncOptions:
     - Validate=false
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: obslytics.dh-prod-argo
+spec:
+  destination:
+    namespace: dh-prod-thanos-extractor
+    server: https://datahub.psi.redhat.com:443
+  project: data-hub
+  source:
+    path: obslytics/overlays/prod
+    repoURL: https://github.com/AICoE/internal-data-hub.git
+    targetRevision: main
+  syncPolicy:
+    automated:
+      prune: true
```

Comment:
I would disable auto-snyc until we can confirm that argocd isn't doing anything unexpected.

Reply:
Ok, so according to the docs, once the syncPolicy is removed auto-sync is disabled.